skimage/data/_binary_blobs.py

```diff
@@ -49,9 +49,6 @@ def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
     >>> blobs = data.binary_blobs(length=256, volume_fraction=0.3)
 
     """
-    # filters is quite an expensive import since it imports all of scipy.signal
-    # We lazy import here
-
```

Comment:
I would keep the empty blank line though.

Reply:
Huh? I'm -0 but that doesn't feel like the default to me.